packages/coordinator/src/screens/signIn/SignInScreen.tsx

```diff
@@ -34,11 +40,31 @@ export default function SignInScreen(props: SignInScreenProps) {
     }
   };
 
+  const resetPassword = async () => {
```

Comment:
Why did you decide to merge the two screens into one?

Remember that there's also a popup to add here after the email is sent. I am not sure it's a good idea to add it to this screen as well

Reply:
It is pretty much the same screen, I didn't want to duplicate it. An other approach could be to make a different component of all the screen structure and use it in each screen, but it look quiet the same to me..

Another benefit of doing it in the same screen is that you don't need to type the email again when you forget the password.

Reply:
What about the missing popup?

![image](https://user-images.githubusercontent.com/1502987/147912440-aa7ffd93-a7af-4b7b-a6d4-b215a810a67c.png)

Anyway, I understand the screens may be very similar, but they may diverge in the future. Sign in screen may have a button to register, Reset password screen can have a "contact help" button.

You could put them in the same folder and share the CSS file if it helps, although I would try to avoid that as well.